readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+
+### About the "Project description" section
+The project description is the most critical part of your README. Describe what the project does, why the reader should care about your project, and how it can help the reader. This section should be succinct, use strong verbs, and help the reader determine whether the project is of use to them.  
+
+Screenshots or videos may also be included in this section to help readers understand your project. Any assets you direct users to should exist in the repo.
+
+You can use the below project description format to start filling in your section. An example is provided.
+
+
+With *(this project)* you can *(verb)* *(noun)*...
+
+*(This project)* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *)this project)* *(verb)* *(noun)*... 
+
+
+---
+
+Example: With the Chronologue project, you can discover astronomical events throughout documented history. This project helps you understand the evolution of space in a streamlined format. Unlike googling and searching for important astronomical events, the Chronologue project outlines historical events with an easy to use and simple interface.
+
+---
+
+### About the "Who is the project for" section
+Identify your project audience and state who can use the project and under what terms. 
+
+### About the "Project dependencies" section
+List any pre-requisites a user needs to interact with your project. Include relevant links to installation instructions or resources. 
+
+### About the "Instructions for using project" section
+Describe in clear steps how to get, install, configure, run, and troubleshoot the project. Use a table for troubleshooting common issues to clearly identify issues and solutions.
+ 
+### About the "Additional documentation" section
+Provide additional documentation for users by including links and brief descriptions. Examples of additional documentation include:
+- Project website
+- Twitter handle(s) of project/project owner(s) 
+- Relevant examples
+- Next steps
+- Features planned
+- Known bugs
+- Documentation files
+- Help command(s)
+ 
+ 
+### About the "How to get help" section
+Provide guidance for users seeking assistance by including links and brief descriptions. Examples of help resources include:
+- Google group/mailing list 
+- Email address(es)
+- IRC, Slack, or Discord channels 
+- Bug trackers
+- Stack overflow
+ 
+### About the "Contributing guidelines" section
+Provide clear instructions on how users can contribute to the project by linking to your contributing guidelines document or by embedding the guidelines in your README.
+
+*Note: Consider the stage your project is at. While most contributing guidelines are provided in a separate document, a smaller project may include guidelines in the README. A Good Docs contributing guidelines template is available. [LINK TO KAYLA'S TEMPLATE].*
+
```

Comment:
**Suggestion (non-blocking, clarity):** Maybe mention they could include it if it's really simple and straightforward, such as something that could be described in a paragraph or a few sentences.